### PR TITLE
typo: web/resource-manager/Microsoft.Web

### PR DESCRIPTION
**Pull request by nschonni**
- API capitilization
- policies -> polcies
- accesible -> accessible
- conenction -> connection
- Contsent -> Consent
- Retrive -> Retrieve
- specificed -> specified
- subcription -> subscription
- multiRool -> multiRole
- Sevice -> Service
- VNets capitalization
- configuraiton -> configuration
- performa -> perform
- BizTalk capitalization
- ceritificate -> certificate
- expriration -> expiration
- Double word "be"
- crendentials -> credentials
- serverfarm -> server farm
- polymophic -> polymorphic
- Double word "to"
- resourse -> resource
- defintions -> definitions
- Succesfully -> Successfully
- Instrunctions -> Instructions
- corelated -> correlated
- Reponse -> Response
- recommedation -> recommendation
- availbility -> availability
- VM's -> VMs
- desciption -> description
- identifider -> identifier
- siteextensions -> site extensions
- sourcecontrol -> source control
- htere -> there
- sku's -> SKUs
- shoud -> should

### CODE REVIEW

File: specification/web/resource-manager/Microsoft.Web/stable/2015-08-01/service.json [hunk 1]
```diff
@@ -5904,7 +5904,7 @@
           "ServerFarms"
         ],
         "summary": "List of metrics that can be queried for an App Service Plan",
-        "operationId": "ServerFarms_GetServerFarmMetricDefintions",
+        "operationId": "ServerFarms_GetServerFarmMetricDefinitions",
```
Comment (jianghaolu) [inline specification/web/resource-manager/Microsoft.Web/stable/2015-08-01/service.json]: I understand this is to fix a typo - but existing SDKs will be broken - you might have to wait to fix this in the next API version.

Comment (nschonni) [reply specification/web/resource-manager/Microsoft.Web/stable/2015-08-01/service.json]: Didn't notice the breaking change. I'll remove it from this PR

**Comment by nschonni** [comment]
@jianghaolu I pulled out the breaking changes and added the rest of the files in the namespace. I filed separate issues for the typos in properties and operation names.

**Comment by nschonni** [comment]
@jianghaolu @naveedaz is there anything else to change for this?

**Review by hovsepm — APPROVED** [review]

**Pull request merged**